Find a functionally equivalent code:android.graphics.Canvas.drawRGB:COMMENT
Method Modifier: public      
Comment:/**
 * Fill the entire canvas' bitmap (restricted to the current clip) with the
 * specified RGB color, using srcover porterduff mode.
 *
 * @param r red component (0..255) of the color to draw onto the canvas
 * @param g green component (0..255) of the color to draw onto the canvas
 * @param b blue component (0..255) of the color to draw onto the canvas
 */

@@ -1,3 +1,3 @@
 {
-    native_drawRGB(mNativeCanvas, r, g, b);
+    drawColor(Color.rgb(r, g, b));
 }

